src/main/java/com/cinema/Application.java

```diff
@@ -1,15 +1,21 @@
 package com.cinema;
 
 import com.cinema.lib.Injector;
+import com.cinema.model.CinemaHall;
 import com.cinema.model.Movie;
+import com.cinema.model.MovieSession;
+import com.cinema.service.CinemaHallService;
 import com.cinema.service.MovieService;
+import com.cinema.service.MovieSessionService;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
 
 public class Application {
     private static Injector injector = Injector.getInstance("com.cinema");
 
     public static void main(String[] args) {
+        // Movies
```

Comment:
let's have the same style for these comments
```suggestion
        //Movies
```

Reply:
done